docs/development_guide.rst

```diff
@@ -3,27 +3,60 @@ Development Guide
 
         *----------------------------- Content in Development -----------------------------*
 
-Development Setup
------------------
+Example RackHD Setup
+--------------------
 
 **Prerequisites**
 
-Install Node.js and npm. **Note:** Currently, node versions 0.10.x are supported. node 0.12 and node 4.0 will be supported in the near future.
+The example setup uses Vagrant and ansible locally. Please install them
+directly on your local machine.
+
+
+
+1. Clone at minimum the rackHD repository. This repository includes both git
+   submodules with all the relevant source code, as well as an example vagrant
+   instance to get an instance up and running locally very easily.
+
+   `git clone https://github.com/RackHD/RackHD`
+
+   Within the example/ directory is a setup script to set up an "example rack"
+   entirely virtually - albiet a very simple one. It creates a virtualbox with
```

Comment:
And entirely virtually I think *is* actually correct, but it probably sounds better to just say "virtually" and strike "entirely"